src/WebExternalSearchProvider.cs

```diff
@@ -179,25 +179,33 @@ public IEnumerable<Clue> BuildClues(ExecutionContext context, IExternalSearchQue
 
             try
             {
-                using (var client = new WebClient())
-                using (var stream = client.OpenRead(orgWebSite.Logo)) //Get Full Quality Image
+                if (!string.IsNullOrEmpty(orgWebSite.Logo?.ToString()) && !Uri.IsWellFormedUriString(orgWebSite.Logo?.ToString(), UriKind.Absolute))
                 {
-                    var inArray = StreamUtilies.ReadFully(stream);
-                    if (inArray != null)
+                    orgWebSite.Logo = new Uri(new Uri(orgWebSite.ResponseUri.AbsoluteUri), orgWebSite.Logo.ToString());
```

Comment:
I don't think this needs to be conditional on orgWebSite.Logo being relative as new Uri will handle both absolute and relative paths